extensions/data-plane/data-plane-azure-storage/src/main/java/org/eclipse/edc/connector/dataplane/azure/storage/DestinationBlobName.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
```

Comment:
please verify headers, year is not correct and also company looks lite to not be the right one